bip-0068.mediawiki

```diff
@@ -0,0 +1,199 @@
+<pre>
+  BIP: 68
+  Title: Consensus-enforced transaction replacement signalled via sequence numbers
+  Author: Mark Friedenbach <mark@friedenbach.org>
+  Status: Draft
+  Type: Standards Track
+  Created: 28-05-2015
+</pre>
+
+==Abstract==
+
+This BIP describes a modification to the consensus-enforced semantics of the sequence number field to enable a signed transaction input to remain invalid for a defined period of time after confirmation of its corresponding output, for the purpose of supporting consensus-enforced transaction replacement features.
+
+==Motivation==
+
+Bitcoin has sequence number fields for each coin a transaction is spending. The original idea appears to have been that the highest sequence number should dominate and miners should prefer it over lower sequence numbers. This was never really implemented, and the half implemented code seemed to be making this assumption that miners would honestly prefer the higher sequence numbers, even if the lower ones were much much more profitable. That turns out to be a dangerous assumption, and so most technical people have assumed that kind of sequence number mediated replacement was useless, because there was no way to enforce "honest" behaviour, as even a few rational (profit maximizing) miners would break that completely. The change described by this BIP provides the missing piece that makes sequence numbers do something significant with respect to enforcing transaction replacement without assuming anything other than profit-maximizing behaviour on the part of miners.
```

Comment:
> for each coin a transaction is spending.

suggest "for each input" (more common terminology among non-core devs, and you use it in the Specification section)

> half implemented

probably should be: "half-implemented"

> was useless, because

Suggest removing the comma.